### DISCUSSION TITLE
Recommended housing with display

### DISCUSSION THREAD

**Discussion by ChKK1963** [probonopd/MiniDexed#116] [Q&A]
Can anyone recommend housing which holds a Pi 4B with the LCD display and rotary encoder? The one on the GitHub start page look nice - cannot find something similiar anywhere ...

Or do I have to get a housing with some extra headroom above like for example https://www.berrybase.de/en/raspberry-pi/raspberry-pi-computer/housing/for-raspberry-pi-4-model-b/geh-228-use-f-252-r-raspberry-pi-4-und-3-5-display-schwarz?c=328 (which has room for the "big" LCD screen but also provides a plastic cover) and then drill the openings for display and encoder?

**Comment by probonopd** [comment]
Mine is 3D printed.

**Reply by probonopd** [reply]
This is roughly how I have assembled my components:

![image](https://user-images.githubusercontent.com/2480569/164023830-ebe5b3d2-2263-4dc0-a9b5-bcdf5c13f80a.png)

My 3D model files are not in a state in which they can be shared, but maybe someone wants to design something. Ideally all snap-fit and without screws.

**Reply by ChKK1963** [reply]
So based on the picture you are not using a dedicated DAC but “only” the 3.5mm jack?

**Reply by probonopd** [reply]
Right now I am still waiting on a PCM5102A based DAC module to arrive. In the meantime, I am using [this hack](https://github.com/probonopd/MiniDexed/wiki/Hardware#hdmi-to-audio). Halfway decent but the DAC is probably still better.
